cha/implementaion/playback/playing.tex

```diff
@@ -0,0 +1,50 @@
+\section{Playing \ac{PCM} Data}
+With the data decoded we also create a replacement for the mediaplayer.\mnote[inline]{why btw? could we not simply have used the one we alrdy had?}
+In preparation for the next part of development, enabling devices to connect to one another and create a multi-device system, we treat the data as an incoming stream rather than local data.
+To handle the \ac{PCM} data stream we use the android class \code{AudioTrack} which allows for streaming of PCM audio buffers to an audio sink.
+
+\mnote[inline]{the following is largely related to how AudioTrack works with the hardware, it would greatly benefit from some implementation related issues and observations}
+The \code{AudioTrack} class does impose a couple of requirements on how we handle the data stream, particularly impacting or playback commands.
+First off \code{AudioTrack} requires that we constantly write to it, if this does not happen it will pause in order to release system resources, this can never happen as it would introduce wake up lag upon the next write action.
+Secondly \code{AudioTrack} works with an internal buffer, this buffer has to be completely filled before it can be flushed to the audio sink, which means the data is send in small buffers at a time.\cite{audiotrack}
```

Comment:
It's essentially explaining the rest of the sentence, its related but completely superfluous